Testscripts/Linux/dpdkUtils.sh

```diff
@@ -167,7 +167,8 @@ function Install_Dpdk () {
 			ssh "${1}" ". utils.sh && install_epel"
 			ssh "${1}" "yum -y groupinstall 'Infiniband Support' && dracut --add-drivers 'mlx4_en mlx4_ib mlx5_ib' -f && systemctl enable rdma"
 			check_exit_status "Install Infiniband Support on ${1}" "exit"
-			ssh "${1}" "(grep -E '7.5|7.6|7.8' /etc/redhat-release) && curl https://partnerpipelineshare.blob.core.windows.net/kernel-devel-rpms/CentOS-Vault.repo > /etc/yum.repos.d/CentOS-Vault.repo"
+			ssh "${1}" "(grep -E '7.5' /etc/redhat-release) && rpm -ivh http://vault.centos.org/7.5.1804/updates/x86_64/Packages/kernel-devel-$(uname -r).rpm"
```

Comment:
why 2 versions?

Reply:
previously rpm that we had was supporting 3 distro versions. I couldn't find 7.8's kernel-devel url. SO I added 2 which I found.

Reply:
If there will be a newer version than 7.5.1804, what does it happen?

Reply:
If there is a new kernel version then it can be handled. 
For new distro version we have to update the link. 
Other way around is we can build rpm and keep it in shared location.

Reply:
Yes, it is not easy to maintain this. Maybe we can have a dict to map distro version to rpm package url? like
(['7.5', 'http://vault.centos.org/7.5.1804/updates/x86_64/Packages/'],['7.6', 'http://vault.centos.org/7.6.1810/updates/x86_64/Packages/'],['8.1','http://vault.centos.org//8.1.1911/BaseOS/x86_64/os/Packages/'])

Reply:
Is it possible to detect automatically? or is it possible to query those numbers between "httpP://vault.centos.org/" and "/updates/x86_64" string from other places? If we go with Lili's suggestion even, we should maintain the strings.